### PR TITLE
GSoD application 2019 blog post

### LINKED ISSUE #98: blog post for GSoD application
Blog post that advertises MDAnalysis as potential Google Season of Docs project.

- [ ] Link to NumFOCUS blog post
- [ ] tweet

Should be up by Sunday 2019-04-21.

## Target audience

Technical writers, who should become interested in working with us.

Language: high level, non-specialists

### About MDAnalysls
- What is MDA about? (What is the problem that we are solving)
- What cool stuff is it used for (science, pictures).

### Motivation
- work on a project that is written by scientists for scientists and has become a widely used and useful tool to many scientists. 
- join a welcoming community
- be taught working with open source tools: version control with Git, continuous integration, open source workflow (pull request/review/merge), doc generation with sphinx

## Projects
What projects do we have?

- Summarize projects and explain importance/motivation.
- Link to our https://github.com/MDAnalysis/mdanalysis/wiki/Google-Season-of-Docs/ page ~(needs updating)~.

### PR DESCRIPTION
**Pull request by orbeckst**
fix #98

### CODE REVIEW

**Comment by kain88-de** [comment]
Is the svg rendered in Jekyll?

**Comment by kain88-de** [comment]
Why is this mixed with gsoc?

**Comment by orbeckst** [comment]
gsoc... hm, need to check what happened with my commit.

**Comment by orbeckst** [comment]
SVG renders fine for me.

**Comment by orbeckst** [comment]
I had branched off wrongly... fixed with a rebase.

**Comment by orbeckst** [comment]
@kain88-de can we publish your post PR #100 _before_ this one and link to it for the tools part?

File: _posts/2019-03-19-gsod2019.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: Google Season of Docs 2019 Application
```
Comment (orbeckst) [inline _posts/2019-03-19-gsod2019.md]: The date of the post needs to be changed – perhaps just set it to the application deadline, *2019-04-23-gsod2019.md*.

(Btw, the date was WRONG... it's April, not March, so it needs to be fixed in any case!!)

**Comment by kain88-de** [comment]
i updated the link to the projects build upon mdanalysis

File: _posts/2019-04-23-gsod2019.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+accomplished scientists and **learn about the science**: *How
+molecules (especially biomolecules such as proteins and DNA) are
+simulated* and *how scientists analyze such simulations in order to
+better understand the world at the microscopic scale*.
+
+
+
+# What is MDAnalysis?
+
+MDAnalysis is a **[Python][] library** for the analysis of particle-based
```
Comment (richardjgowers) [inline _posts/2019-04-23-gsod2019.md]: maybe expand and make this sentence more accessible?  "these particles represent the motion of atoms which make up our bodies and the natural world"?

File: _posts/2019-04-23-gsod2019.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+simulations. 
+
+MDAnalysis is also a **community** of hundreds of users and developers. As
+a community, we adhere to our [Code of Conduct][].
+
+The **goal of MDAnalysis** is to make it easy for users to analyze data
+that are produced by simulations (primarily [molecular dynamics][]
+simulations) that run on some of the largest super-computers in the
+world. We accomplish this goal by providing an abstract and
```
Comment (richardjgowers) [inline _posts/2019-04-23-gsod2019.md]: Maybe simplify here.  "by providing a toolkit of programming building blocks which provide an object oriented interface to..."

**Comment by richardjgowers** [comment]
@orbeckst I made a few comments, but looks good

File: _posts/2019-04-23-gsod2019.md (hunk 3 above)
Comment (kain88-de) [reply _posts/2019-04-23-gsod2019.md]: Can we start to skip the object oriented? Yeah we use the idom. I'd rather not limit it to only using this idom. There are parts where we do not use OO but rather composition.

**Comment by orbeckst** [comment]
@richardjgowers @kain88-de please feel free to change – I won't be able to do anything today.

**Comment by kain88-de** [comment]
I'm good.

File: _posts/2019-04-23-gsod2019.md (hunk 2 above)
Comment (orbeckst) [reply _posts/2019-04-23-gsod2019.md]: MDAnalysis is a **[Python][] library** for the analysis of computer
simulations of many-body systems at the molecular scale.

File: _posts/2019-04-23-gsod2019.md (hunk 3 above)
Comment (orbeckst) [reply _posts/2019-04-23-gsod2019.md]: addressed

**Comment by orbeckst** [comment]
@kain88-de @richardjgowers thanks!

**Pull request merged**